code/test/data/report-input/000006/status.yaml

```diff
@@ -1,6 +1,6 @@
 dandiset_version: 9c6ab8750946cdac16c62057ec1f76f9ce954fe1
 last_run: 2022-12-15 10:59:47.526056-05:00
-nassets: 42
+nassets: 0
```

Comment:
is that a changed semantic -- that it is not total number of assets present but rather - how many were "considered" and since no tests ran - nothing considered, thus 0?

Reply:
The file in question is just a manually-generated input file for the test run of the `report` command.  This field is ignored by the command, but I felt it better for the sample inputs to be "semantically self-consistent", i.e., for `nassets` to match the number of assets listed in the file.